doc-content/enterprise-only/installation/nfs-server-configure-proc.adoc

```diff
@@ -1,29 +1,32 @@
 [id='nfs-server-configure-proc_{context}']
 
-= Configuring an NFS server
+= Configuring an NFS version 4 server
 
-A shared file system is required for a {CENTRAL} clustered environment and each client node must have access to the shared file system.
-
-You must deploy and configure an NFS version 4 server.
+An NFS version 4 server with a shared file system is required for a {CENTRAL} clustered environment and each client node must have access to the shared file system.
 
 .Procedure
 
 . Configure a server to export NFS version 4 shares. For instructions about exporting NFS shares on Red Hat Enterprise Linux, see _Exporting NFS shares_ in https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_file_systems/index[_Managing file systems_]. For more information about creating the NFS server, see https://access.redhat.com/solutions/1355233[_How to configure NFS in RHEL 7_].
 
-. On the server, create an `/opt/kie/data` share with the `rw,sync,no_root_squash` options by adding the following line to the `/etc/exports` file:
+. On the server, open the `/etc/exports` file in a text editor.
+
+. Add the following line to the `/etc/exports` file where `<HOST_LIST>` is a space-separated list of IP addresses and options of hosts that are allowed connect to the server:
 +
-[subs="attributes,verbatim,macros"]
+[source]
 ----
-/opt/kie/data *(rw,sync,no_root_squash)
+/opt/kie/data <HOST_LIST>
 ----
 +
-In this example, `/opt/kie/data` is the shared folder, `*` are the IP addresses allowed to connect to the NFS server, and `(rw,sync,no_root_squash)` are the minimum options required for NFS. For example:
+For example:
 +
+[source]
 ----
-/opt/kie/data 192.268.1.0/24(rw,sync,no_root_squash)
+/opt/kie/data  192.268.1.0/24(rw,sync)  192.268.1.1/24(no_root_squash)
 ----
 +
+Doing this creates an `/opt/kie/data` share with the `rw,sync,no_root_squash` options which are the minimum options required for NFS.
```

Comment:
[Style] Remove "Doing" and maybe use the restrictive clause?

s/Doing this creates an `/opt/kie/data` share with the `rw,sync,no_root_squash` options which are the minimum options required for NFS./This creates an `/opt/kie/data` share with the `rw,sync,no_root_squash` minimum options that are required for NFS.